### PR TITLE
Display the magnitude of linear and rotationnal speed

### PR DESCRIPTION
**Pull request by ShuffleWire**
**Description**
Allows to easily monitor the displacement speed of a robot with changing direction

### CODE REVIEW

**Comment by stefaniapedrazzi** [comment]
Linear and angular velocities are 3D vectors.
Adding a forth value like it is done in this PR is wrong and would just confuse users.

**Comment by ShuffleWire** [comment]
What about this way ?
image

**Comment by stefaniapedrazzi** [comment]
This would be acceptable.

Please check your code style before asking for a new review (const variables, variables names, etc.) and make sure you also add an entry in the ChangeLog.

File: docs/reference/changelog-r2021.md [hunk 1]
```diff
@@ -20,6 +20,7 @@ Released on June, Xth, 2021.
     - Add an example that shows an integration of OpenAI Gym with Webots ([#2711](https://github.com/cyberbotics/webots/pull/2711)).
     - Added a simple room with a Nao robot ([#2701](https://github.com/cyberbotics/webots/pull/2701)).
   - New Features:
+    - Display magnitude of velocity/rotational speed vectors, to help monitoring robot displacement ([#3209](https://github.com/cyberbotics/webots/pull/3209)).
```
Comment (stefaniapedrazzi) [inline docs/reference/changelog-r2021.md:23]: This is an enhancement and not a new major feature. Could you please move it in the `Enhancements` section?
Moreover we should use the correct terminology as displayed in the UI:
```suggestion
    - Display magnitude of linear and rotational velocity vectors in the [Field Editor](../guide/the-scene-tree.md#field-editor), to help monitoring robot displacement ([#3209](https://github.com/cyberbotics/webots/pull/3209)).
```

**Review by stefaniapedrazzi — CHANGES_REQUESTED** [review]
Thank you.
This contribution is fine.
Could you please just adjust the change log entry?

**Review by stefaniapedrazzi — APPROVED** [review]
Thank you

**Pull request merged**